src/test/java/spoon/test/template/testclasses/InvocationTemplate.java

```diff
@@ -15,15 +15,15 @@ void invoke() {
 	
 	@Local
 	public InvocationTemplate(CtTypeReference<?> ifaceType, String methodName) {
-		this.ifaceType = ifaceType;
-		this.methodName = methodName;
+		this._IFace = ifaceType.getSimpleName();

```

Comment:
We both are right and spoon supports both ways.
A) if one needs to rename type the s/he use parameter value of type String
B) if one needs to substitute type by another type, then s/he use CtTypeReference